packages/solo/test/test-home.js

```diff
@@ -1,12 +1,6 @@
 /* global process */
 
-// `test.after.always` does not yet seem compatible with ses-ava
-// See https://github.com/endojs/endo/issues/647
-// TODO restore
-// import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
-import '@agoric/swingset-vat/tools/prepare-test-env.js';
-// eslint-disable-next-line import/no-extraneous-dependencies
-import test from 'ava';
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
```

Comment:
I made this change to be consistent with the other code.  I do think we'd be better off with `import test from 'ava'` but that's [out of scope](https://github.com/Agoric/agoric-sdk/pull/5774#issuecomment-1186539828)).

> Why are you using the @foo/something for here … but not in other places?

I didn't change any import paths so not really my decision. But I think the relative ones are because what's imported is coming from the same package.

Reply:
ok. Resolving.